From 97977a1534ef27247acc9549f5433ef288f208eb Mon Sep 17 00:00:00 2001
From: Peter Ujfalusi <peter.ujfalusi@ti.com>
Date: Mon, 23 Sep 2013 11:25:04 +0300
Subject: [PATCH 228/752] ARM/dts: am335x-evmsk: Remove num-serializer
 property for mcasp node

The num-serializer property is redundant and the driver no longer needs it.

Signed-off-by: Peter Ujfalusi <peter.ujfalusi@ti.com>
---
 arch/arm/boot/dts/am335x-evmsk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/am335x-evmsk.dts b/arch/arm/boot/dts/am335x-evmsk.dts
index b87ce7f..354c0e5 100644
--- a/arch/arm/boot/dts/am335x-evmsk.dts
+++ b/arch/arm/boot/dts/am335x-evmsk.dts
@@ -471,7 +471,7 @@
 
 		op-mode = <0>;          /* MCASP_IIS_MODE */
 		tdm-slots = <2>;
-		num-serializer = <16>;
+		/* 16 serializer */
 		serial-dir = <  /* 0: INACTIVE, 1: TX, 2: RX */
 			0 0 1 2
 			0 0 0 0
-- 
1.7.10.4

